website/docs/docs/on-premises/release-process.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Release process
+---
+
+Minor version releases are performed at 1-2 month intervals. Patch version releases are performed as required.
```

Comment:
Can't think of anything meaningful to say about major release, because we've never done one.